lib/active_merchant/billing/gateways/eway_managed.rb

```diff
@@ -232,33 +232,33 @@ def soap_request(arguments, action)
         # eWay demands all fields be sent, but contain an empty string if blank
         post = case action
                when 'QueryCustomer'
-                  arguments
+                 arguments
                when 'ProcessPayment'
-                  default_payment_fields.merge(arguments)
+                 default_payment_fields.merge(arguments)
                when 'CreateCustomer'
-                  default_customer_fields.merge(arguments)
+                 default_customer_fields.merge(arguments)
                when 'UpdateCustomer'
-                  default_customer_fields.merge(arguments)
+                 default_customer_fields.merge(arguments)
                end
 
         xml = Builder::XmlMarkup.new :indent => 2
-          xml.instruct!
-          xml.tag! 'soap12:Envelope', {'xmlns:xsi' => 'http://www.w3.org/2001/XMLSchema-instance', 'xmlns:xsd' => 'http://www.w3.org/2001/XMLSchema', 'xmlns:soap12' => 'http://www.w3.org/2003/05/soap-envelope'} do
-            xml.tag! 'soap12:Header' do
-              xml.tag! 'eWAYHeader', {'xmlns' => 'https://www.eway.com.au/gateway/managedpayment'} do
-                xml.tag! 'eWAYCustomerID', @options[:login]
-                xml.tag! 'Username', @options[:username]
-                xml.tag! 'Password', @options[:password]
-              end
+        xml.instruct!
```

Comment:
Shouldn't this be indented?

Reply:
No; there's no `do` on the previous line, and this is not a block. It's honestly really misleading for exactly that reason as-is.